### PR TITLE
fix: remove usage of currentInstance, re-implement indicator count up…

### PR DESCRIPTION
**Pull request by mcstover**
…date mechanism

For this update, I chose to create a new story with controls to demonstrate the indicatory updating behavior and reInit functionality by programmatically removing slides. Remove at least 2 slides while running the story to verify the new behavior.

### CODE REVIEW

File: @kiva/kv-components/vue/KvCarousel.vue [hunk 1]
```diff
@@ -14,7 +14,7 @@
 				:key="index"
 				class="tw-flex-none tw-relative"
 				role="group"
-				:aria-label="`slide ${index} of ${componentSlotKeys.length}`"
+				:aria-label="`slide ${index + 1} of ${componentSlotKeys.length}`"
```
Comment (mcstover) [inline @kiva/kv-components/vue/KvCarousel.vue:17]: Noticed this was off by 1. "0 of 8" slides when creating the story below.

File: @kiva/kv-components/vue/KvCarousel.vue [hunk 2]
```diff
@@ -193,6 +192,19 @@ export default {
...
+
+		/**
+		 * Returns number of slides in the carousel
+		 *
+		 * @returns {Number}
+		 */
+		const slideIndicatorListLength = () => {
+			const indicator = embla.value ? embla.value.scrollSnapList().length : 0;
+			slideIndicatorCount.value = indicator;
+			console.log(indicator);
```
Comment (dyersituations) [inline @kiva/kv-components/vue/KvCarousel.vue:204]: Remove `console.log`?

**Review by dyersituations — APPROVED** [review]

File: @kiva/kv-components/vue/stories/KvCarousel.stories.js [hunk 3]
```diff
@@ -164,6 +164,78 @@ export const MultipleLoanCards2 = () => ({
...
+			this.slideIndicatorCount = this?.$refs?.myCarouselRef?.slideIndicatorCount;
+		});
+	},
+	methods: {
+		removeSlide() {
+			// Fetch slides + remove one
+			const slides = this?.$refs?.myCarouselRef?.embla?.slideNodes();
+			const slideToRemove = slides.pop();
+			this.removedSlides.push(slideToRemove);
+			this?.$refs?.myCarouselRef?.embla?.containerNode()?.removeChild(slideToRemove);
```
Comment (roger-in-kiva) [inline @kiva/kv-components/vue/stories/KvCarousel.stories.js:228]: **this?** is correct?

Comment (mcstover) [reply @kiva/kv-components/vue/stories/KvCarousel.stories.js:228]: lolz. It's Functional but def not needed in this context

Comment (mcstover) [reply @kiva/kv-components/vue/stories/KvCarousel.stories.js]: removed

**Review by emuvente — APPROVED** [review]
👍  Looks good! Glad to not need getCurrentInstance anymore!

**Pull request merged**